src/cmd/dev.go

```diff
@@ -237,7 +237,7 @@ var devSha256SumCmd = &cobra.Command{
 }
 
 var devFindImagesCmd = &cobra.Command{
-	Use:     "find-images [ PACKAGE ]",
+	Use:     "find-images [ DIRECTORY ]",
```

Comment:
sneaking this in a bit, but noticed this while writing the ZEPs. This runs on a directory not a package. Switching this to use the same verbiage used in `zarf package create`.